docs/tokens/esdt-tokens.mdx

```diff
@@ -450,17 +446,15 @@ _For more details about how arguments have to be encoded, check [here](/develope
 
 Following this transaction, the total supply of tokens is increased by the new supply specified in the Data field, and the manager receives that amount of tokens into their balance.
 
-This operation requires that the option `canMint` is set to `true` for the token.
-
-Alternatively, an account with the `ESDTRoleLocalMint` role set can perform a local mint:
+An account with the `ESDTRoleLocalMint` role set can perform a local mint:
 
 ```rust
 LocalMintTransaction {
     Sender: <address with ESDTRoleLocalMint role>
     Receiver: <same as sender>
     Value: 0
     GasLimit: 300000
-    Data: "ESDTLocalMint" +
+    Data: "ESDTRoleLocalMint" +
```

Comment:
I think it was ok before.

Reply:
Corect, funcția este `ESDTLocalMint` iar rolul este `ESDTLocalMint`